content/productivity/pink-it/plugin.js

```diff
@@ -0,0 +1,529 @@
+
+/*
+	made for DF Archon Ares v0.1 Round 2
+	https://mirror.xyz/dfarchon.eth/hns4_mHAdUvPVDLrnCVxxlg1MrxpQMDgPTHsBgLXhRw
+	
+	features:
+	- can automatically destroy all planets inside the pink circle (using dropped bomb from the pink ship)
+	- shows a list of all pink circles in view and their owners/operators
+	- click any planet to see how big a pink circle would be if a bomb were dropped on it
+*/
+
+const pluginName = "Pink It";
+
+const simulate = false; // for testing purposes this can be set to true (if true it wont destroy planets)
+
+const cSpace = "&nbsp;"
+const minCircleSizePx = 5;
+const minDrawPlanetRadius = 20;
+const circleThickness = 2;
+const PI2 = Math.PI * 2;
+
+const pinkCircleRadius = 2000;
+const minPinkLevel = 3; // cant pink level below 3
+
+let choosenPlanets = [];
+let deadPlanets = [];
+let dieingPlanet = null;
+let stopPinkIt = false;
+let excludeMyPlanets = true;
+let excludeOwnedPlanets = false;
+let pinkItIsRunning = false;
+let butPinkIt = null;
+let butPinkItIsReady = false;
+let myPinkZone = null;
+let hlPinkZoneCounter = 0;
+const hlPinkZoneCounterMax = 200;
+let lastSelectedPlanet = null;
+let tablePinkZoneList = null;
+let intervUpdateTablePZList = null;
+let lastWorldCoords = null;
+let pzHoveringInList = null;
+
+import { getPlayerColor } from "https://cdn.skypack.dev/@darkforest_eth/procedural";
```

Comment:
```suggestion
import { getPlayerColor } from "https://cdn.skypack.dev/@dfares/procedural";
```

1. imports should be at the top of the file
2. import the from `@dfares` rather than  the `@darkforest_eth` registry.